akka-actor-typed-tests/src/test/java/jdocs/akka/typed/StashDocTest.java

```diff
@@ -78,28 +78,33 @@ public DBError(RuntimeException cause) {
     }
 
     private final ActorContext<Command> context;
-    private final StashBuffer<Command> buffer = StashBuffer.create(100);
+    private final StashBuffer<Command> buffer;
     private final String id;
     private final DB db;
 
-    private DataAccess(ActorContext<Command> context, String id, DB db) {
+    private DataAccess(
+        ActorContext<Command> context, StashBuffer<Command> buffer, String id, DB db) {
       this.context = context;
+      this.buffer = buffer;
       this.id = id;
       this.db = db;
     }
 
     public static Behavior<Command> create(String id, DB db) {
       return Behaviors.setup(
-          ctx -> {
-            ctx.pipeToSelf(
-                db.load(id),
-                (value, cause) -> {
-                  if (cause == null) return new InitialState(value);
-                  else return new DBError(asRuntimeException(cause));
-                });
-
-            return new DataAccess(ctx, id, db).init();
-          });
+          ctx ->
```

Comment:
hmm, wouldn't the "normal" indent in Java be:

```
return Behaviors.setup(ctx -> {
      // etc
});
```

Saves one level of indent which is good for the docs that has width issues.